introduction.md

```diff
@@ -1,2 +1,3 @@
-JavaScript
+*Git
+*JavaScript
 BLack929 introducing to the dojo
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?